### PR TITLE
Opt: Remove CommonUniformElimination from -O and -Os (for now)

### PR DESCRIPTION
**Pull request by greg-lunarg**
It is causing crashes for some drivers. Will try to re-enable it once
existing drivers are better able to deal with it.

### CODE REVIEW

File: source/opt/optimizer.cpp [hunk 1]
```diff
@@ -77,7 +77,8 @@ Optimizer& Optimizer::RegisterPerformancePasses() {
       .RegisterPass(CreateBlockMergePass())
       .RegisterPass(CreateLocalMultiStoreElimPass())
       .RegisterPass(CreateInsertExtractElimPass())
-      .RegisterPass(CreateCommonUniformElimPass())
+      // Currently causing driver crashes
```
Comment (dnovillo) [inline source/opt/optimizer.cpp]: Could you reference the issue? It's #946, IIRC.

File: source/opt/optimizer.cpp [hunk 2]
```diff
@@ -92,7 +93,8 @@ Optimizer& Optimizer::RegisterSizePasses() {
...
```
Comment (dnovillo) [inline source/opt/optimizer.cpp]: Likewise here.  Thanks.

File: source/opt/optimizer.cpp (hunk 1 above)
Comment (greg-lunarg) [reply source/opt/optimizer.cpp]: Done.

File: source/opt/optimizer.cpp (hunk 2 above)
Comment (greg-lunarg) [reply source/opt/optimizer.cpp]: Done.

**Comment by dnovillo** [comment]
Rebased, tested and pushed as commit d86c7ce808

**Pull request closed without merging**